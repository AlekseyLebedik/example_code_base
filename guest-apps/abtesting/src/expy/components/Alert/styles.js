import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
  container: {
    padding: '5px',
    borderRadius: '4px',
    border: '3px transparent solid',
    backgroundColor: '#FFF',
    marginBottom: '1rem',
  },
  titleContainer: {
    display: 'flex',
    alignItems: 'end',
  },
  title: {
    fontWeight: 700,
    marginBottom: '0px',
  },
  description: {
    marginTop: '5px',
    marginLeft: '26px',
    marginBottom: '0px',
    color: '#000',
  },
  icon: {
    marginLeft: '5px',
    marginRight: '7px',
  },
  denied: {
    borderLeftColor: theme.palette.statuses.denied.main,
  },
  approved: {
    borderLeftColor: theme.palette.statuses.approved.main,
  },
  pending: {
    borderLeftColor: theme.palette.statuses.pending.main,
  },
  info: {
    borderLeftColor: theme.palette.statuses.info.main,
  },
}));
