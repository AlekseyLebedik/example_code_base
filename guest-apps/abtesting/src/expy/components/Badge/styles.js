import { makeStyles } from '@material-ui/core/styles';
import { themeStyles } from '../../../shared/theme';

export const useStyles = makeStyles(() => {
  const { proposal, ready, cancelled, live, done } = themeStyles;

  return {
    badge: {
      display: 'inline',
      fontSize: '0.75rem',
      marginBottom: '0',
      backgroundColor: '#ddd',
      borderRadius: '4px',
      fontWeight: '700',
      letterSpacing: '0.5px',
      marginRight: '5px',
      padding: '0.25rem 0.75rem',
      fontFamily: 'Roboto, sans-serif',
    },
    bullet: {
      '&::before': {
        backgroundColor: '#666',
        borderRadius: '7px',
        content: '""',
        display: 'inline-block',
        height: '7px',
        marginRight: '7px',
        width: '7px',
      },
    },
    white: {
      backgroundColor: '#FFFFFF',
    },
    basic: {
      backgroundColor: 'rgba(90, 104, 112, 0.1)',
    },
    proposal: {
      backgroundColor: proposal.light,
      '&::before': {
        backgroundColor: proposal.main,
      },
    },
    ready: {
      backgroundColor: ready.light,
      '&::before': {
        backgroundColor: ready.main,
      },
    },
    cancelled: {
      backgroundColor: cancelled.light,
      '&::before': {
        backgroundColor: cancelled.main,
      },
    },
    live: {
      backgroundColor: live.light,
      '&::before': {
        backgroundColor: live.main,
      },
    },
    done: {
      backgroundColor: done.light,
      '&::before': {
        backgroundColor: done.main,
      },
    },
  };
});
