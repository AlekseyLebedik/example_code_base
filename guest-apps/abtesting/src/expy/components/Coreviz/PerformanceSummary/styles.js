import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
  container: {
    backgroundColor: '#FFF',
    borderRadius: '4px',
    marginBottom: theme.spacing(2),
    padding: theme.spacing(2),
  },
  badge: {
    padding: '0.25rem 0.75rem',
    borderRadius: '4px',
    display: 'inline-block',
  },
  circle: {
    width: '12px',
    height: '12px',
    borderRadius: '50%',
    backgroundColor: theme.palette.grey[700],
  },
  green: {
    backgroundColor: theme.palette.statuses.approved.light,
  },
  grey: {
    backgroundColor: theme.palette.grey[300],
    padding: '0.525rem 0.825rem',
  },
  red: {
    backgroundColor: theme.palette.statuses.denied.light,
  },
  title: {
    fontWeight: 700,
  },
  details: {
    display: 'flex',
    justifyContent: 'space-between',
    alignItems: 'center',
    marginBottom: theme.spacing(1),
    '&:last-of-type': {
      marginBottom: 0,
    },
  },
  nameContainer: {
    display: 'flex',
    alignItems: 'center',
  },
  name: {
    marginBottom: 0,
    marginLeft: theme.spacing(1),
  },
  value: {
    fontWeight: 700,
    marginBottom: 0,
  },
}));
