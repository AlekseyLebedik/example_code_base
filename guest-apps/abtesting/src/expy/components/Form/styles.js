import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
  select: {
    width: '100%',
    '& .abtesting-MuiSelect-select:focus': {
      backgroundColor: '#FFF',
    },
  },
  selectItem: {
    width: '100% !important',
  },
  btnContainer: {
    width: '100%',
    display: 'flex',
    justifyContent: 'center',
    marginTop: theme.spacing(4),
    marginBottom: theme.spacing(3),
  },
  submitBtn: {
    marginRight: theme.spacing(1),
    backgroundColor: theme.palette.primary.main,
  },
  errorText: {
    fontSize: '0.8571428571428571rem',
    color: theme.palette.error.main,
    marginLeft: '14px',
    marginTop: '3px',
  },
}));
