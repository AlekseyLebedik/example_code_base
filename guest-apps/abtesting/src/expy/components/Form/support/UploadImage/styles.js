import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
  dropzone: {
    textAlign: 'center',
    padding: '20px',
    borderRadius: '0',
    border: `1px dashed ${theme.palette.grey[400]}`,
    backgroundColor: '#FFFFFF',
    outline: 'none',
    transition: 'border .24s ease-in-out',
    width: '100%',
    '&:focus': {
      borderColor: theme.palette.primary.main,
    },
  },
  heading: {
    color: theme.palette.grey[800],
    marginBottom: theme.spacing(1),
    fontSize: '18px',
  },
  description: {
    marginTop: theme.spacing(1),
    marginBottom: theme.spacing(1),
    color: theme.palette.grey[800],
  },
  warning: {
    color: theme.palette.grey[800],
    fontWeight: 700,
    marginTop: theme.spacing(1),
  },
  imageContainer: {
    backgroundColor: '#FFF',
    border: `1px solid ${theme.palette.grey[400]}`,
    textAlign: 'center',
  },
  imageContainerBtn: {
    display: 'flex',
    justifyContent: 'center',
    marginTop: theme.spacing(3),
    marginBottom: theme.spacing(3),
  },
  deleteImage: {
    backgroundColor: theme.palette.error.main,
    '&:hover': {
      backgroundColor: theme.palette.error.dark,
    },
  },
  image: {
    marginTop: theme.spacing(3),
    maxWidth: '100%',
  },
  pdf: {
    marginTop: theme.spacing(3),
  },
}));
