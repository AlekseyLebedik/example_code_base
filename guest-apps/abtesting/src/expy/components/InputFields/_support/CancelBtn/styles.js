import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
  button: {
    background: 'transparent',
    cursor: 'pointer',
    border: 'transparent',
    padding: `2px ${theme.spacing(0.5)}px`,
  },
}));
