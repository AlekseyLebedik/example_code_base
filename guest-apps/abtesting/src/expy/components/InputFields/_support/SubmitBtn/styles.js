import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
  button: {
    marginRight: theme.spacing(1),
    marginLeft: theme.spacing(1),
    background: theme.palette.primary.main,
    color: '#fff',
    cursor: 'pointer',
    border: 'transparent',
    borderRadius: '4px',
    padding: `2px ${theme.spacing(0.5)}px`,
    '&:disabled': {
      opacity: '0.6',
      cursor: 'not-allowed',
    },
  },
}));
