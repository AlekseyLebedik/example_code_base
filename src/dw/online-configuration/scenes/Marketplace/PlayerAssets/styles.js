import { makeStyles } from '@material-ui/core/styles';

export const muiStyles = () => ({
  rootButton: { fontSize: '20px', zIndex: '1' },
  disabledButton: { color: 'gray' },
  hidenButton: { visibility: 'hidden' },
  rootCheckbox: { zIndex: '1' },
  actionButtonContainer: {
    width: '55px',
  },
});

export const useStyles = makeStyles(muiStyles);
