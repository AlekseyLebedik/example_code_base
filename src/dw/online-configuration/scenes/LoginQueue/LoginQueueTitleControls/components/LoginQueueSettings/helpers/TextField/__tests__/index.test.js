import React from 'react';
import { shallow } from 'enzyme';
import TextFieldQueues from '../index';
import styles from '../../../index.module.css';
import NumberFormatCustom from '../../CustomFormatInputBox';

describe('LoginQueueSettings - TextField', () => {
  it('Text field check', () => {
    const textfieldqueue = shallow(
      <TextFieldQueues
        className={styles.loginQueueFormInput}
        value={300000}
        disabled={false}
        onChange={() => null}
        key="maxCCU"
        name="maxCCU"
        InputProps={{
          inputComponent: NumberFormatCustom,
        }}
      />
    );
    expect(textfieldqueue).toMatchSnapshot();
  });
});
