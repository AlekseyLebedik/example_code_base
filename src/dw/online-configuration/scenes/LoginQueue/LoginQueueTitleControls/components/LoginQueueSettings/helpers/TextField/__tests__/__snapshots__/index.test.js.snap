// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginQueueSettings - TextField Text field check 1`] = `
<WithStyles(ForwardRef(TextField))
  InputProps={
    Object {
      "inputComponent": [Function],
    }
  }
  className="loginQueueFormInput"
  classes={
    Object {
      "root": "WithStyles(ForwardRef(TextField))-root-1",
    }
  }
  disabled={false}
  name="maxCCU"
  onChange={[Function]}
  value={300000}
/>
`;
